apps/api/src/app/events/usecases/parse-event-request/parse-event-request.usecase.ts

```diff
@@ -238,9 +240,7 @@ export class ParseEventRequest {
       throw new UnprocessableEntityException('Environment not found');
     }
 
-    const statelessWorkflowAllowed = environment.name !== 'Production';
```

Comment:
@scopsy made an earlier change to allow for stateless Production executions, but we didn't remove this check.